lib/vsc/utils/script_tools.py

```diff
@@ -408,30 +753,32 @@ def main(self):
         self.fulloptions.epilogue(f"{msg} complete", self.thresholds)
 
 
+@deprecated_class("Base your scripts on the CLIBase class instead")
 class NrpeCLI(CLI):
```

Comment:
why are these deprecated?

Reply:
Because I want to switch to the new stack, obviously. It's not that they'll stop working, but I'd prefer no new scripts use these.